### ISSUE TITLE
Support start tiles for MS Edge

### ISSUE THREAD

**Issue by danielthepope** [danielthepope/trntxt#31]
In order to get a custom tile icon onto the Windows 10 start menu using MS Edge, do I need different meta tags or will the IE ones be fine?

At the moment the meta tags aren't included because Edge has a different UA string than IE. Important since Windows 10 Mobile will use Edge.

**Comment by patrickkettner** [comment]
Hey @danielthepope - super neat service!

The same meta tags work in Edge/Win10. Here 

![image](https://cloud.githubusercontent.com/assets/465414/9265467/8eadeda6-4201-11e5-925a-9c02eeb11df0.png)

[Here](https://github.com/h5bp/html5-boilerplate/pull/1697#issuecomment-105641354) is a bit more information, specifically

> These are still supported in Edge...

and

> I'd like to see us deprecate browserconfig.xml and associated meta tags in favor of using a standard JSON manifest. But I don't know of imminent plans to do so quite yet and I believe there's still some work to do on the manifest specification itself

**Comment by danielthepope** [comment]
Hey @patrickkettner, thanks for the info! I was trying to find something about browserconfig.xml in the MSDN docs but there's not very much for Edge yet. I'll make sure all browsers see the manifest.json (I thought it was just a Chrome thing rather than a proper standard).
Looks like in the example from your screenshot, Edge has gone out to look for trntxt.uk/browserconfig.xml even though it's not actually specified in the meta tag (yet).

**Comment by patrickkettner** [comment]
the manifest is not quite a proper standard - but we are hoping that it becomes one asap.

but yeah, it will grab the browserconfig automaticall

**Comment by danielthepope** [comment]
I updated my site so that the meta tag pointing to the custom browserconfig appears to Edge browsers. I tried pinning the tile to my start menu when running locally, but for some reason Edge didn't even make a request for browserconfig.xml. Not so good for testing. I then deployed it anyway and pinned from the live site and it worked!
![image](https://cloud.githubusercontent.com/assets/5173183/9285355/0079c21c-42dd-11e5-840d-6f7eb1f473c8.png)
I think I need to adapt the icon a bit, so that the app name is easier to read!
